packages/azure-http-specs/specs/azure/resource-manager/resources/mockapi.ts

```diff
@@ -516,13 +922,169 @@ Scenarios.Azure_ResourceManager_Resources_ExtensionsResources_delete = passOnSuc
+      status: 200,
+      body: json({
+        value: [validSubscriptionExtensionsResource],
+      }),
+    },
+    handler: (req: MockRequest) =>
+      requestHandler(req, "list", SUBSCRIPTION_SCOPE_URI, validSubscriptionExtensionsResource),
+    kind: "MockApiDefinition",
+  },
+  {
+    uri: "/:resourceUri/providers/Azure.ResourceManager.Resources/extensionsResources",
+    method: "get",
+    request: {
+      pathParams: {
+        resourceUri: SUBSCRIPTION_SCOPE_URI.substring(1),
+      },
+      query: {
+        "api-version": "2023-12-01-preview",
+      },
+    },
+    response: {
+      status: 200,
+      body: json({
+        value: [validSubscriptionExtensionsResource],
+      }),
+    },
+    handler: (req: MockRequest) =>
+      requestHandler(req, "list", SUBSCRIPTION_SCOPE_URI, validSubscriptionExtensionsResource),
+    kind: "MockApiDefinition",
+  },
+  {
+    uri: "/:resourceUri/providers/Azure.ResourceManager.Resources/extensionsResources",
+    method: "get",
+    request: {
+      pathParams: {
+        resourceUri: TENANT_SCOPE_URI,
+      },
+      query: {
+        "api-version": "2023-12-01-preview",
+      },
+    },
+    response: {
+      status: 200,
+      body: json({
+        value: [validTenantExtensionsResource],
+      }),
+    },
+    handler: (req: MockRequest) =>
+      requestHandler(req, "list", TENANT_SCOPE_URI, validTenantExtensionsResource),
+    kind: "MockApiDefinition",
+  },
+  {
+    uri: "/:resourceUri/providers/Azure.ResourceManager.Resources/extensionsResources",
+    method: "get",
+    request: {
+      pathParams: {
+        resourceUri: RESOURCE_SCOPE_URI,
+      },
+      query: {
+        "api-version": "2023-12-01-preview",
+      },
+    },
+    response: {
+      status: 200,
+      body: json({
+        value: [validResourceExtensionsResource],
+      }),
+    },
+    handler: (req: MockRequest) =>
+      requestHandler(req, "list", RESOURCE_SCOPE_URI, validResourceExtensionsResource),
+    kind: "MockApiDefinition",
+  },
+  {
+    uri: "/:resourceUri/providers/Azure.ResourceManager.Resources/extensionsResources",
+    method: "get",
+    request: {
+      pathParams: {
+        resourceUri: RESOURCE_SCOPE_URI.substring(1),
+      },
+      query: {
+        "api-version": "2023-12-01-preview",
+      },
+    },
+    response: {
+      status: 200,
+      body: json({
+        value: [validResourceExtensionsResource],
+      }),
+    },
+    handler: (req: MockRequest) =>
+      requestHandler(req, "list", RESOURCE_SCOPE_URI, validResourceExtensionsResource),
     kind: "MockApiDefinition",
   },
 ]);
 
 Scenarios.Azure_ResourceManager_Resources_ExtensionsResources_listByScope = passOnSuccess([
   {
-    uri: "/subscriptions/:subscriptionId/resourceGroups/:resourceGroup/providers/Azure.ResourceManager.Resources/extensionsResources",
+    uri: "//subscriptions/:subscriptionId/resourceGroups/:resourceGroup/providers/Azure.ResourceManager.Resources/extensionsResources",
```

Comment:
Why the change here? IMHO since we are using `withServiceKeys`, there should be no `passOnSuccess` anymore.

Reply:
Removed redundant parts by [6d3a56e](https://github.com/Azure/typespec-azure/pull/2203/commits/6d3a56edf3f754deec6eec21e4d52d8d97974e94)